portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,76 @@
+          <li><a href="index.html#projects">Projects</a></li>
+          <li><a href="index.html#timeline">Timeline</a></li>
+          <li><a href="index.html#about">About Me</a></li>
+          <li><a href="index.html#home">Home</a></li>
+        </ul>  
+      </div>
+      <div class="gallery-title">
+        <h3>My Gallery</h3>
+      </div>
+    </header>
+    </header>
+    <!-- Gallery Section --> 
+    <section id="gallery" class="light"> 
+      <div class="container">
+        <div class="row"> 
+            <div class="column gallery quarter">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+            </div>
+            <div class="column gallery quarter">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+            </div>  
+            <div class="column gallery quarter">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+            </div>
+            <div class="column gallery quarter">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+                <img src="/images/sunset.jpg" style="width:100%">
+                <img src="/images/ryan.jpg" style="width:100%">
+            </div>
+        </div>
+      </div>
+    </section>
+  </body>
+</html> 
```

Comment:
Please add a newline at the end of files. (A lot of editors do this by default, sorry you have to do this manually 😕)

Reply:
oh okay, done